cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/SessionManager.java

```diff
@@ -165,6 +176,10 @@ synchronized void closeSession(HttpServletRequest request) {
     if (info != null && info.lastFuture().isDone()) {
       LOG.info("Closing session {}", session);
       session.invalidate();
+      _sessionLifetimeTimer.update(System.nanoTime() - info.requestTime(), TimeUnit.NANOSECONDS);
+      if (!hasError && info.executionTime() > 0) {
```

Comment:
If we actually check whether there is error or not (i.e. `hasError`), shouldn't we call the `_requestExecutionTimer` as `_successfulRequestExecutionTimer`?